README.md

```diff
@@ -40,31 +40,28 @@ The class components in the current version are `NumberGuessingGame` and `GuessC
-- [ ] Create `currentGuess` and `setCurrentGuess` state variables using the `useState` hook and initialize the value to an empty string. Set the `value` property for the input element to refer to this state value. (Make sure you include `useState` as an import). 
-- [ ] Create a `handleInputChange` function within the component that updates the `currentGuess` state value when the user change the value in the input. Set the `onChange` property for the input element to refer to this function
+- [ ] Create new state variable named `currentGuess` with setter `setCurrentGuess` and default value of an empty string. Set the `value` property for the input element to refer to this state value. (Make sure to import `useState`)
+- [ ] Create a `handleInputChange` function within the component that updates the `currentGuess` state value when the user changes the value in the input. Set the `onChange` property for the input element to refer to this function.
 - [ ] Create a `onSubmitGuess` function that calls the `onGuess` prop with the `currentGuess` value converted to a number and also resets the `currentGuess` to an empty string when it is called. Set the `onClick` property on the button to refer to this function.
-- [ ] If you still have the old class version around as `GuessControlOld` delete it
+- [ ] If you still have the old class version around as `GuessControlOld`, delete it.
 
 ### Convert `NumberGuessingGame`
 
-- [ ] Open `src\NumberGuessingGame.js`
+- [ ] Open `/src/NumberGuessingGame.js`
 - [ ] Rename the current `NumberGuessingGame` class to `NumberGuessingGameOld` if you want to keep it a reference while converting the code
-- [ ] Create a new function component called `NumberGuessingGame`
+- [ ] Create a new function component called `NumberGuessingGame` with parameter `props`.
 - [ ] Copy the logic and return value from the render function in the class component to be in the new function component. Remove any references to `this.` since those will be replaced with new references.
-- [ ] Create 3 state variables and their setters for `numberToGuess`, `numberOfGuesses`, and `latestGuess` and initialize them to the same values from the class component version. (Make sure you include `useState` as an import).
-- Create a `handleGuess` function that will be passed in to the `GuessControl` component as the `onGuess` prop. This function should take the guess as an argument and set the `latestGuess` state with the guess and increment the `numberOfGuesses` state. 
+- [ ] Create 3 state variables and their setters for `numberToGuess`, `numberOfGuesses`, and `latestGuess` and initialize them to the same values from the class component version. (Make sure to import `useState`)
+- Create a `handleGuess` function that will be passed in to the `GuessControl` component as the `onGuess` prop. This function should take the guess as an argument and set the `latestGuess` state with the guess (converted to a Number) and increment the `numberOfGuesses` state. 
```

Comment:
I missed the step of converted `guess` string to number at first, so I think we should be more explicit about that step

Reply:
Yeah, I can see that. I debated how much of the step by step to provide that was just walking them through the exact same code that wouldn't change between the class vs function versions of the components. I leaned towards focusing on the aspects that would change, but we can call this step out more clearly. I did try to help with this as well by including a comment in the original code explaining why this step was being taken.

I will make this more explicit and add even more to the comment in the source code about what is happening.